content/style-injector.js

```diff
@@ -94,40 +106,56 @@ function createStyleInjector({compare, setStyleContent, onUpdate}) {
     style.el.remove();
   }
 
-  function update({id, code}) {
+  function _update({id, code}) {
     const style = table.get(id);
     if (style.code === code) return;
     style.code = code;
     // workaround for Chrome devtools bug fixed in v65
-    // https://github.com/openstyles/stylus/commit/0fa391732ba8e35fa68f326a560fc04c04b8608b
-    let oldEl;
-    if (CHROME < 3321) {
-      oldEl = style.el;
+    if (isChromePre65) {
+      const oldEl = style.el;
       oldEl.id = '';
-      style.el = createStyle(id);
+      style.el = createStyle(id, code);
       oldEl.parentNode.insertBefore(style.el, oldEl.nextSibling);
-      style.el.disabled = !enabled;
+      oldEl.remove();
+    } else {
+      style.el.textContent = code;
+      style.el.sheet.disabled = !enabled;
```

Comment:
What is the difference between `el.sheet.disabled` and `el.disabled`?

Reply:
There's none so I'll remove `sheet`.